src/main/java/hudson/tasks/test/AbstractTestResultAction.java

```diff
@@ -287,6 +316,84 @@ public TestResult findCorrespondingResult(String id) {
         return Collections.emptyList();
     }
 
+    /**
+     * Start for Test Result Trends.
+     * From here onwards start the code area responsible for generating various test result trends.
+     */
+
+    public hudson.tasks.junit.TestResult loadXml(){return null;}
+
+    /**
+     * A method for getting the list of packages for all levels of hierarchy.
+     * @return Array of packages of all hierarchies.
+     */
+    public String[] getProjectList(){
```

Comment:
There should be a space after method name`getProjectList() {` . Please indent properly at all the places, Your IDE would be having a default indentation format, please make use of it.